cypress/support/fragments/check-out-actions/check-out-actions.js

```diff
@@ -26,11 +26,11 @@ export default {
     ]);
   },
 
-  async checkUserInfo({ barcode, personal }, patronGroup = '0') {
+  async checkUserInfo({ barcode, username }, patronGroup = '0') {
```

Comment:
This method can be used in other autotests. So changing this method can affect them. Please don't change this method in this way, you can add your own new method to the test this case.
Also I have doubts if the username is shown always. Need to recheck the cases.